metron-deployment/roles/kibana/README.md

```diff
@@ -1,35 +0,0 @@
-Kibana 4
---------
-
-This role installs Kibana along with the default Metron Dashboard.
-
-### FAQ
-
-#### How do I change Metron's default dashboard?
```

Comment:
I'm inclined (possibly by my own self interest) to make it a follow on jira that gets resolved either before or after 745.  745 takes care of this file (which is actually why I thought of it).

I definitely agree on using the same file though, but I'm not sure off the top of my head how much refactoring happens there.